File: Modul-1/Soal-10/PRAK110-2410817210008-AhmadUlyani.c
#include <stdio.h>
#include <math.h>

int main ()
{
    // Panjang dalam sentimeter
    int alas = 5;
    int tinggi = 12;
    int sisimiring = sqrt(alas * alas + tinggi * tinggi);
    int keliling = alas + tinggi + sisimiring;
    int luas = (alas * tinggi) / 2;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("\nJawab :\n");
    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %d cm\n", sisimiring);
    printf("Sisi C = %d cm\n", alas);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm\n", luas);

    return 0;
}

File: Modul-1/Soal-10/PRAK110-2410817210008-AhmadUlyani.py
import math
alas = 5
tinggi = 12
sisimiring = math.sqrt(alas * alas + tinggi * tinggi)
keliling = alas + tinggi + sisimiring
luas = (alas * tinggi) / 2

print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print("")
print("Jawab :")
print(f"Sisi A = {tinggi} cm")
print(f"Sisi B = {sisimiring:.0f} cm")
print(f"Sisi C = {alas} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm")